packages/react-components/src/components/chart/trendCursor/mouseEvents/useTrendCursorsEvents.ts

```diff
@@ -66,6 +78,7 @@ const useTrendCursorsEvents = ({
             tcId: `trendCursor-${uuid()}`,
             timestamp: timestampInMs,
           });
+          setIsInCursorAddModeRef.current(false); // change cursor back to regular from cross hair (hotkeys)
```

Comment:
this should be added after the else loop...

Reply:
Also, why forcefully stop the add mode ? like user is still holding the t ... and we are kicking them out of the add mode ?